src/fsharp/ExtensionTyping.fsi

```diff
@@ -12,7 +12,7 @@ open FSharp.Core.CompilerServices
 open FSharp.Compiler.AbstractIL.IL
 open FSharp.Compiler.Text
 
-module internal ExtensionTyping =
```

Comment:
This should not be public.  Please update the baselines for FCS surface area so we could see properly what's been added